src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -594,7 +596,7 @@ partial class BoundIncrementOperator : IIncrementExpression
 
     partial class BoundBadExpression
     {
-        protected override OperationKind ExpressionKind => OperationKind.None;
+        protected override OperationKind ExpressionKind => OperationKind.BadExpression;
```

Comment:
The usual feedback has been to surface names used in the bound trees, at least when those names are common across languages. I've used the term "Bad" for this sort of thing for so long that for me it seems natural, but I'm happy to use another term.

"Invalid" is rejected by some policy checkers which recommend "disabled" or some such instead.